supporter-product-data/src/test/scala/com/gu/Fixtures.scala

```diff
@@ -0,0 +1,8 @@
+package com.gu
+
+import scala.io.Source
+
+//noinspection SourceNotClosed
+object Fixtures {
+  def loadQueryResults: String = Source.fromURL(getClass.getResource("/query-results.csv")).mkString
```

Comment:
marginal, but it's one small file so the indirection may be unnecessary
```suggestion
  def loadQueryResults: String =
    """
Account.IdentityId__c,Subscription.GifteeIdentityId__c,RatePlan.Id,ProductRatePlan.Id,ProductRatePlan.Name,Subscription.TermEndDate,Subscription.RedemptionCode__c,Subscription.GifteeIdentityId__c
13380875,,2c92a00777427d35017744760ace4c77,2c92a0ff5345f9220153559d915d5c26,Percentage,2022-01-27,,
13380875,,2c92a00777427d35017744760aef4c7c,2c92a0fb4edd70c8014edeaa4e972204,Digital Pack Annual,2022-01-27,,
3204264,105320827,2c92a00b7742896c017744780f57375e,2c92a00773adc09d0173b99e4ded7f45,Digital Subscription One Year Fixed,2022-01-28,gd12-aw655rhh,105320827
""".trim
```

Reply:
I put this in when the file was much bigger, I'm going to leave it now